### PR TITLE
Added support for setting column widths per sheet

### LINKED ISSUE #88: How to set column width for certain sheet only
Hello,

Is there any options to set column width for certain sheet only ?

### PR DESCRIPTION
**Pull request by mlntn**
For issue #88

### CODE REVIEW

File: src/Writer/Common/Entity/Sheet.php [hunk 1]
```diff
@@ -160,4 +164,45 @@ public function getAutoFilter(): ?AutoFilter
     {
         return $this->autoFilter;
     }
+
+    /**
+     * @param positive-int ...$columns One or more columns with this width
+     */
+    final public function setColumnWidth(float $width, int ...$columns): void
```
Comment (Slamdunk) [inline src/Writer/Common/Entity/Sheet.php]: Overall this PR looks good, but I don't like the complexity under the hood of this function signature.
I'd prefer force the user call this multiple times more than force ourself to maintain such a complex function content.
```suggestion
    final public function setColumnWidth(float $width, int $column): void
```

**Review by Slamdunk — CHANGES_REQUESTED** [review]
:muscle:

File: src/Writer/Common/Entity/Sheet.php (hunk 1 above)
Comment (mlntn) [reply src/Writer/Common/Entity/Sheet.php]: Thanks for reviewing!

This decision was based on an existing method: https://github.com/openspout/openspout/blob/90ceeefef3750dd2afab95689ed2107dd576905e/src/Writer/Common/AbstractOptions.php#L30 Do you want me to deviate from that?

Comment (Slamdunk) [reply src/Writer/Common/Entity/Sheet.php]: Didn't remember that (aweful) code, that's for pointing it out

**Review by Slamdunk — CHANGES_REQUESTED** [review]
Please run CS Fix and we are done

**Review by Slamdunk — APPROVED** [review]

**Pull request merged**